src/haddock/modules/defaults.yaml

```diff
@@ -149,5 +149,5 @@ less_io:
   group: "execution"
   explevel: easy
   incompatible:
-    value: true
-    mode: batch
+    true:
```

Comment:
This format change will need to fixed at https://github.com/i-VRESSE/workflow-builder/blob/f59b3d1f20a04dd1afe87427eb47a9e3316eb53d/packages/haddock3_catalog/generate_haddock3_catalog.py#L128-L144

Reply:
outch...
Maybe not the best solution then...
But nevertheless I believe it would allow more flexibility in the way we can handle incompatibilities

Reply:
does not hurt that bad, but it will take some time to adjust. Mostly the comment was a reminder for myself.